core/mondoo-azure-security.mql.yaml

```diff
@@ -1387,44 +1385,47 @@ queries:
   - uid: mondoo-azure-security-ensure-activity-log-alert-exists-for-create-update-delete-network-security-group
     title: Ensure that activity log alerts exist for the commands Create, Update, and Delete Network Security Group
     impact: 80
+    filters: |
+      asset.platform == "azure"
```

Comment:
Shouldn't this just run on a azure subscription?

Reply:
Yes sure, all of them run on Azure subscription

Reply:
platform of azure is the platform for the subscriptions